mern-ecommerce/controllers/order.js

```diff
@@ -61,3 +61,17 @@ exports.updateOrderStatus = (req, res) => {
     }
   );
 };
+
+exports.removeOrderById = (req, res) => {
+  Order.deleteOne({ _id: req.params.orderId })
+    .exec()
+    .then(result => {
+      if (result.n === 0) {
+        return res.status(404).json({ message: "Order not found" });
+      }
+      res.status(200).json({ message: "Order successfully removed" });
+    })
+    .catch(err => {
+      res.status(500).json({ error: errorHandler(err) });
+    });
```

Comment:
In the provided code snippet, there is only a closing bracket '});' which is not a complete code block. Therefore, I will review the entire file context.

In 'mern-ecommerce/controllers/order.js', the function names are not following a consistent naming convention. Some functions are in camelCase (e.g., orderById) while others are in underscore notation (e.g., updateOrderStatus). It would be better to stick to a single convention throughout the codebase.

In 'mern-ecommerce/models/product.js', the productSchema definition is correct, but it would be better to separate the schema definition and model creation into different files for better organization and reusability.

In 'mern-ecommerce/routes/auth.js', the route names are not following a consistent naming convention. Some routes are in camelCase (e.g., signup) while others are in underscore notation (e.g., signout). It would be better to stick to a single convention throughout the codebase.

In 'mern-ecommerce/server.js', the route imports are not following a consistent naming convention. Some routes are in camelCase (e.g., orderRoutes) while others are in underscore notation (e.g., braintreeRoutes). It would be better to stick to a single convention throughout the codebase. Also, the 'orderRoute' import is missing an 's' at the end, it should be 'orderRoutes'.